src/gpu/gpu_deconvolution_list.cpp

```diff
@@ -34,6 +34,9 @@
 #include "gpu/generic/sycl/ref_deconvolution.hpp"
 #endif
 
+#include "gpu/generic/convolution_deconvolution.hpp"
+#include "gpu/intel/ocl/convolution_deconvolution.hpp"
```

Comment:
`gpu/intel` directory is not included in the build if gpu vendor is not intel. The header should always be guarded.

Reply:
Removed this include, as it was a duplicate of the guarded one.